sender/sender.go

```diff
@@ -20,11 +20,14 @@ type BySMS interface {
 
 type ByChat interface {
 	ByPush
+	To(to string, cc ...string)
+}
+
+type ByApi interface {
+	ByPush
 }
 
 type ByPush interface {
-	UID() string
 	WithBody(body string)
```

Comment:
Did you intend to remove UID from the interface?

Reply:
And take it `To` was moved to ByChat as it is only used by that inherited interface?

Reply:
Nothing uses the UID.
I moved To up because it didn't make sense for the API interface.